internal/core/api.rs

```diff
@@ -324,6 +324,31 @@ pub enum SetRenderingNotifierError {
     AlreadySet,
 }
 
+/// This struct represents a persistent handle to a window and implements the traits
+/// for exposing raw window handles.
```

Comment:
maybe a link to the Window::window_handle function to tell how to construct it. 
Also linking to the the traits it implements can be usefull

Reply:
Good idea.